src/KalturaUploader.tsx

```diff
@@ -94,12 +99,19 @@ const KalturaUploaderInternal = ({
   endpoint,
   ks,
   partnerId,
+  callback,
 }: KalturaUploaderProps): JSX.Element => {
   const [state, dispatch] = useReducer(reducer, { id: "start" });
   const [mediaType, setMediaType] = useState<MediaTypeOptions>(
     MediaTypes.video
   );
 
+  useEffect(() => {
+    if (state.id === "complete") {
+      callback([state.entry]);
```

Comment:
Hmmm, interesting that the ESLint React Hooks plugin did not complain about this usage of `state.entry`. :thinking: 

Ah! We don't have that plugin setup yet. I'll do that in another branch. But you might want to add the dependency here.